include/seqan/align_split/align_split_interface.h

```diff
@@ -370,16 +403,16 @@ int _splitAlignmentImpl(Gaps<TContigSeqL> & gapsContigL,
         }
     }
 
-    // std::cerr << "bestPrefixLength = " << bestPrefixLength << "\n";
-
-    // std::cerr << "split store left ";
-    // for (unsigned i = 0; i < length(scoutStateL.splitScore); ++i)
-    //     fprintf(stderr, " %3d", scoutStateL.splitScore[i]);
-    // std::cerr << "\n";
-    // std::cerr << "split store right";
-    // for (unsigned i = 0; i < length(scoutStateR.splitScore); ++i)
-    //     fprintf(stderr, " %3d", scoutStateR.splitScore[i]);
-    // std::cerr << "\n";
+//     std::cerr << "bestPrefixLength = " << bestPrefixLength << "\n";
+//
+//     std::cerr << "split store left ";
+//     for (unsigned i = 0; i < length(scoutStateL.splitScore); ++i)
+//         fprintf(stderr, " %3d", scoutStateL.splitScore[i]);
+//     std::cerr << "\n";
+//     std::cerr << "split store right";
+//     for (unsigned i = 0; i < length(scoutStateR.splitScore); ++i)
+//         fprintf(stderr, " %3d", scoutStateR.splitScore[i]);
+//     std::cerr << "\n";
 
```

Comment:
Seems like different commenting schemes from different editors. I'll change that.